src/cpp/src/visual_language/pipeline.cpp

```diff
@@ -80,7 +81,12 @@ class VLMPipeline::VLMPipelineImpl : public VLMPipelineBase{
         if (m_is_npu) {
             embedder_device = "CPU";
             utils::KVDesc kv_desc;
+            if (auto it = lm_properties.find("MAX_PROMPT_LEN"); it == lm_properties.end()) {
+                // Let default value be 3072 for context length, if not specified by user.
+                lm_properties["MAX_PROMPT_LEN"] = 3072;
+            }
```

Comment:
Is this change mandatory? How our existing tests passed without one? Having a 3K prompt _by default_ puts a substantial pressure on footprint and performance.

Reply:
No, it is not! Tests work without it. It is here just to allow some bigger text inputs or, for example. two images. Do you think it is better to return 1024 limit?